### PR TITLE
Changed participant select element from select to number input.

### PR DESCRIPTION
**Pull request by ANschoolacc**
Changed participant select element from select to number input. Added new validation in js for the participants number input. Removed some superfluous code from booking.js that created options for participants select element before. Changed a little styling for time and participants input.

### CODE REVIEW

**Review by maxlin94 — CHANGES_REQUESTED** [review]
Tested with live server, the input works as it should and looks good, except it doesnt shrink on smaller screen sizes like the other inputs. 

It has the same max-width but no width: 80% like the others do.

File: src/booking.scss [hunk 1]
```diff
@@ -1,299 +1,300 @@
...
+    &__e-mail-input {
+        width: 80%;
+        max-width: 300px;
+        height: 30px;
+        padding: 5px;
+        border-radius: 5px;
+        border: solid lightgrey;
+    }
+
+    &__select {
```
Comment (maxlin94) [inline src/booking.scss:183]: Missing width: 80%

Comment (ANschoolacc) [reply src/booking.scss:183]: Good catch! Fixed that in the latest commit.

**Review by maxlin94 — APPROVED** [review]
Nice, everything looks good now!

**Pull request merged**